server/plugins/pfed-sso/src/main/java/com/walmartlabs/concord/server/plugins/pfedsso/SsoConfiguration.java

```diff
@@ -0,0 +1,151 @@
+package com.walmartlabs.concord.server.plugins.pfedsso;
+
+/*-
+ * *****
+ * Concord
+ * -----
+ * Copyright (C) 2017 - 2019 Walmart Inc.
+ * -----
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =====
+ */
+
+import com.walmartlabs.ollie.config.Config;
+import org.eclipse.sisu.Nullable;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.inject.Singleton;
+import java.io.Serializable;
+import java.time.Duration;
+
+@Named
+@Singleton
+public class SsoConfiguration implements Serializable {
+
+    @Inject
+    @Config("sso.pfed.enabled")
+    private boolean enabled;
+
+    @Inject
+    @Config("sso.pfed.priority")
+    private Integer priority;
```

Comment:
Can be `int` here, the code treats it as `int` anyway.
+ the same thing in the same class in the iam-sso module.

Reply:
changed to `int`